src/main/webapp/app/core/interceptor/artemis-version.interceptor.ts

```diff
@@ -67,12 +67,18 @@ export class ArtemisVersionInterceptor implements HttpInterceptor {
      * - the first condition was ever true since the app loaded (aka last reload)
      *
      * We need to have this second option because the "checkForUpdate()" call sometimes starts to return false after a while, even though we didn't reload / update yet.
+     * And if service workers are not available we can't actually check for updates, so we have to rely on ever having seen a different version number in a request
+     *
+     * @param hasUpdate if it is known that there is an update, only relevant if service workers are not available
      *
      * @private
      */
-    private checkForUpdates() {
+    private checkForUpdates(hasUpdate: boolean) {
+        // don't spam errors when service workers are not available, instead rely on the Content-Version header of responses
+        const update = this.updates.isEnabled ? this.updates.checkForUpdate() : Promise.resolve(hasUpdate);
+
         // first update the service worker
-        this.updates.checkForUpdate().then((updateAvailable: boolean) => {
+        update.then((updateAvailable: boolean) => {
```

Comment:
I think we'd need to bypass ``this.updates.activateUpdate()`` as well in L95, right? That would trigger an error as well
So if the sw is not enabled, we should just reload the page instead of telling the sw to update